<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardDetailWidget</class>
 <widget class="QWidget" name="CardDetailWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>972</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_id">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spin_id">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999990</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_dbid">
           <property name="text">
            <string>DB ID:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spin_dbid">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_code">
           <property name="text">
            <string>Code:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="line_code">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="line_name">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_rarity">
           <property name="text">
            <string>Rarity:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="line_rarity">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_type">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="line_type">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_class">
           <property name="text">
            <string>Class:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="line_class">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_race">
           <property name="text">
            <string>Race:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="line_race">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_set">
           <property name="text">
            <string>Set:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="line_set">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_spellschool">
           <property name="text">
            <string>Spellschool:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="line_spellschool">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_faction">
           <property name="text">
            <string>Faction:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="line_faction">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_cost">
           <property name="text">
            <string>Cost:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSpinBox" name="spin_cost">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_collectible">
           <property name="text">
            <string>Collectible:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QCheckBox" name="check_collectible">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_elite">
           <property name="text">
            <string>Elite:</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QCheckBox" name="check_elite">
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_tag">
           <property name="text">
            <string>Tag:</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="line_tag">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_mechanic">
           <property name="text">
            <string>Mechanic:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="line_mechanic">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_text">
           <property name="text">
            <string>Text:</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="line_text">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_artist">
           <property name="text">
            <string>Artist:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLineEdit" name="line_artist">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QLabel" name="label_flavor">
           <property name="text">
            <string>Flavor:</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QLineEdit" name="line_flavor">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_attack">
           <property name="text">
            <string>Attack:</string>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QSpinBox" name="spin_attack">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="label_health">
           <property name="text">
            <string>Health:</string>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QSpinBox" name="spin_health">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="label_overload">
           <property name="text">
            <string>Overload:</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QSpinBox" name="spin_overload">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="22" column="0">
          <widget class="QLabel" name="label_durability">
           <property name="text">
            <string>Durability:</string>
           </property>
          </widget>
         </item>
         <item row="22" column="1">
          <widget class="QSpinBox" name="spin_durability">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QLabel" name="label_spelldamage">
           <property name="text">
            <string>Spelldamage:</string>
           </property>
          </widget>
         </item>
         <item row="23" column="1">
          <widget class="QSpinBox" name="spin_spelldamage">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="24" column="0">
          <widget class="QLabel" name="label_armor">
           <property name="text">
            <string>Armor:</string>
           </property>
          </widget>
         </item>
         <item row="24" column="1">
          <widget class="QSpinBox" name="spin_armor">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9999999</number>
           </property>
          </widget>
         </item>
         <item row="25" column="0">
          <widget class="QLabel" name="label_questreward">
           <property name="text">
            <string>Questreward:</string>
           </property>
          </widget>
         </item>
         <item row="25" column="1">
          <widget class="QLineEdit" name="line_questreward">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
